package.json

```diff
@@ -140,8 +139,8 @@
     "http-link-header": "^0.8.0",
     "inquirer": "^3.3.0",
     "intl": "^1.2.5",
-    "intl-messageformat": "^2.2.0",
-    "intl-messageformat-parser": "^1.4.0",
+    "intl-messageformat": "^4.4.0",
```

Comment:
wow that's a big jump, didn't we just add this a year ago? I thought intl rules would be more stable by now 😆

Reply:
> wow that's a big jump, didn't we just add this a year ago? I thought intl rules would be more stable by now 😆

yeah, it was unchanged for two years until three months ago, at which point the versions started exploding :)

https://www.npmjs.com/package/intl-messageformat?activeTab=versions

I mentioned above, but I didn't pick the latest version because of the change in the AST format in 5.0.0 that didn't seem worth updating for given that everything works today.